app/views/errors/error.html.slim

```diff
@@ -0,0 +1,8 @@
+section.headline.headline-error
+  .l-wrapper.headline-body.error
+    h1.error-code= @error_code
+    = image_tag 'lupa.png', class: 'error-img'
```

Comment:
why using "lupa.png" (spanish) instead of "magnifying_glass.png" or "lens.png" ??/